src/pheval/analyse/analysis.py

```diff
@@ -37,18 +40,26 @@ def _read_standardised_result(standardised_result_path: Path) -> dict:
 
 def parse_pheval_gene_result(pheval_gene_result: pd.DataFrame) -> [RankedPhEvalGeneResult]:
     """Parse PhEval gene result into RankedPhEvalGeneResult dataclass."""
-    ranked_gene_results = []
-    for _index, result in pheval_gene_result.iterrows():
-        ranked_gene_results.append((RankedPhEvalGeneResult(*result)))
-    return ranked_gene_results
+    return [
+        RankedPhEvalGeneResult(**row._asdict())
+        for row in pheval_gene_result.itertuples(index=False)
+    ]
 
 
 def parse_pheval_variant_result(pheval_variant_result: pd.DataFrame) -> [RankedPhEvalVariantResult]:
     """Parse PhEval variant result into RankedPhEvalVariantResult dataclass."""
-    ranked_variant_results = []
-    for _index, result in pheval_variant_result.iterrows():
-        ranked_variant_results.append(RankedPhEvalVariantResult(*result))
-    return ranked_variant_results
+    return [
+        RankedPhEvalVariantResult(**row._asdict())
+        for row in pheval_variant_result.itertuples(index=False)
+    ]
+
+
+def parse_pheval_disease_result(pheval_disease_result: pd.DataFrame) -> [RankedPhEvalDiseaseResult]:
```

Comment:
For my own learning, is `[RankedPhEvalDiseaseResult]` the same as `List[RankedPhEvalDiseaseResult]`?

Reply:
Yep, it's the same

Reply:
Thank you!